Example/GiniVision/UIViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  UIViewController.swift
+//  GiniVision_Example
+//
+//  Created by Enrique del Pozo Gómez on 5/8/18.
+//  Copyright © 2018 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+import GiniVision
+
+extension UIViewController {
```

Comment:
I'm not sure it makes sense for this functionality to be in a View controller extension. It's not connected to view controllers.

Reply:
It is just an example. There are different ways to implement this.